### PR TITLE
Generating Test Data through Integration Tests

### PR DESCRIPTION
**Pull request by SantamRC**
To execute run,
`python print-data.py`

 Test Cases Generation Progress:

- [ ] Reducer.cpp
- [ ] Slice,cpp
- [ ] Index.cpp
- [x] Identities.cpp
- [x] Content.cpp
- [ ] Files in layoutbuilder
- [ ] Files in array

### CODE REVIEW

File: print-data.py [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+    record = False
+    x = open(os.path.join(CURRENT_DIR, "src", "libawkward", "Index.cpp"), "r")
+    lines = x.readlines()
+    for line in lines:
+        if re.search(r"struct\sError\serr\s=\skernel::\w+", line):
+            record = True
+        elif re.search(r"[)];", line) and record == True:
+            record = False
+            new_file += line
+            new_file += '\tstd::cout<<"awkward_Index_to_Index64:";for(int i=0;i<length();i++){std::cout <<ptr()[i]<<",";}std::cout<<":";printMe();std::cout<<":"<<length_<<std::endl;\n'
```
Comment (ianna) [inline print-data.py]: @SantamRC - have you updated the PR? This line gives numerous  compilation errors:
```
/Users/yana/Projects/IRIS-Fellow/Santam/new-plan/awkward-1.0/src/libawkward/Index.cpp:294:81: error: type 'std::shared_ptr<int64_t>' (aka 'shared_ptr<long long>') does not provide a call operator
        std::cout<<"awkward_Index_to_Index64:";for(int i=0;i<length();i++){std::cout <<ptr()[i]<<",";}std::cout<<":";printMe();std::cout<<":"<<length_<<std::endl;
                                                                                       ^~~
```

**Review by ianna — CHANGES_REQUESTED** [review]
please, make sure the branch is up-to-date with your local changes. Thanks!

**Comment by ianna** [comment]
@SantamRC - are you still working on this PR? Please, close it if not. Thanks!

**Comment by SantamRC** [comment]
@ianna I am working on getting the data for the remaining kernel functions right now. Using this PR for it. Will close after it's done.

**Pull request closed without merging**